test/vdirel-test.el

```diff
@@ -85,5 +85,14 @@
               '(("Damien Cassou <me@foo.com>" . ("Damien Cassou" "me@foo.com"))
                 ("Damien Cassou <me@bar.eu>" . ("Damien Cassou" "me@bar.eu")))))))
 
+
+
+(describe "contact-no-name"
+          (it "contacts without FN or F should return empty string"
```

Comment:
can you please move this `it` to the `describe` called "knows how to extract a contact's fullname" and rename it to "when there is neither FN nor F"?